### PR TITLE
Added support for nessus:pro data.

### PR DESCRIPTION
**Pull request by VatsalJagani**

### CODE REVIEW

**Review by mahirchavda — APPROVED** [review]

File: cyences_app_for_splunk/bin/cs_product_list.py [hunk 1]
```diff
@@ -322,6 +322,19 @@ def build_source_reviewer_search(by, values, first_call=True):
...
         ],
     },
+    {
+        "name": "Nessus",
+        "macro_configurations": [
+            {
+                "macro_name": "cs_nessus",
+                "label": "Nessus Data",
+                "search_by": "sourcetype",
+                "search_values": "nessus:pro:vuln,nessus:pro:plugin,nessus_json",
```
Comment (hardikhdholariya) [inline cyences_app_for_splunk/bin/cs_product_list.py:332]: is this source=nessus:pro:plugin required? seems we are not using in cyences

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 2]
```diff
@@ -6218,6 +6219,60 @@ search = `cs_tenable_vuln` | dedup tenable_uuid, vul_id \
...
+| rename host as nessus_collected_by, network_name as nessus_network_name, state as nessus_state \
+| fillnull value="-" nessus_network_name \
+| table time, indextime, product_name, product_uuid, ip, mac_address, hostname, user, nessus_collected_by, nessus_fqdn, nessus_netbios, nessus_os, nessus_state, nessus_network_name, created_at, first_scan_time, first_seen, has_agent, has_plugin_results, last_authenticated_scan_date, last_licensed_scan_date, last_seen, nessus_uuid \
+| cyencesdevicemanager operation="addentries" \
+| append [| inputlookup cs_nessus_inventory] \
+| stats latest(time) as time, latest(*) as * by nessus_uuid \
+| eval _key=nessus_uuid | outputlookup cs_nessus_inventory
+action.cyences_notable_event_action.products = nessus
```
Comment (hardikhdholariya) [inline cyences_app_for_splunk/default/savedsearches.conf:6246]: Nessus

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 3]
```diff
@@ -6218,6 +6219,60 @@ search = `cs_tenable_vuln` | dedup tenable_uuid, vul_id \
...
+search = `cs_nessus_vuln` | dedup nessus_uuid, vul_id \
+| eval time=_time, indextime=_indextime, product_name="nessus", product_uuid=nessus_uuid \
+| fillnull ip, hostname, mac_address, user value="" \
+| table time, indextime, _time, product_name, product_uuid, hostname, user, ip, mac_address, nessus_uuid, vul_id, vul_name, vul_description, vul_severity, vul_severity_id, vul_state, last_fixed, last_found, vul_cve, vul_solution, vul_cpe, vul_family, vul_has_patch, vul_in_the_news, vul_risk_factor, vul_synopsis, vul_type, vul_version, vul_protocol, vul_port \
+| cyencesdevicemanager operation="addentries" \
+| fields - time \
+| append [| inputlookup cs_nessus_vuln] \
+| dedup nessus_uuid, vul_id sortby -_time \
+| outputlookup cs_nessus_vuln
+action.cyences_notable_event_action.products = nessus
```
Comment (hardikhdholariya) [inline cyences_app_for_splunk/default/savedsearches.conf:6273]: Nessus

File: cyences_app_for_splunk/bin/cs_product_list.py (hunk 1 above)
Comment (VatsalJagani) [reply cyences_app_for_splunk/bin/cs_product_list.py]: right, removed

File: cyences_app_for_splunk/default/savedsearches.conf (hunk 2 above)
Comment (VatsalJagani) [reply cyences_app_for_splunk/default/savedsearches.conf]: Done

File: cyences_app_for_splunk/default/savedsearches.conf (hunk 3 above)
Comment (VatsalJagani) [reply cyences_app_for_splunk/default/savedsearches.conf]: Done

**Review by hardikhdholariya — APPROVED** [review]

**Pull request merged**